### ISSUE TITLE
[Bug]: Google sheets insert new row query copies the formatting from the previous row

### ISSUE THREAD

**Issue by ayushpahwa** [appsmithorg/appsmith#11772]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Whenever we insert a row in google sheets, the formatting of the row above is copied below. This is based on the query raised by a user on [discord](https://discord.com/channels/725602949748752515/951084742219005982). 
<img width="1199" alt="image" src="https://user-images.githubusercontent.com/8526215/157667831-b7b17ffd-382f-4d9c-a122-16882ebc6200.png">

### Steps To Reproduce

1. Create a google sheets query and add a header row to the sheet.
2. Give header any formatting and insert rows from the appsmith query.
3. The new rows will copy the formatting from the last row added above.

### Public Sample App

https://app.appsmith.com/applications/6229dabaf782567d61f60980/pages/6229dabaf782567d61f60982/edit/saas/google-sheets-plugin/api/6229dabaf782567d61f60985

### Version

Cloud

**Comment by Nikhil-Nandagopal** [comment]
So the main issue here is to just not copy the formatting of the header

**Comment by zzgvh** [comment]
Hi, I reported this issue via the Discord support channel. I'm not 100% sure, but I think this bug is a regression. I've had the same code running for quite some time and I think it was when I upgraded two or three weeks ago that this bug appeared. This might not be so much help though, since I upgraded from 1.6.4 to 1.6.12... :man_shrugging: But maybe it can be of some little help when resolving the issue.
